connectors/sources/oracle.py

```diff
@@ -344,20 +344,23 @@ def get_default_configuration(cls):
                 "order": 9,
                 "type": "str",
                 "value": DEFAULT_PROTOCOL,
+                "ui_restrictions": ["advanced"],
```

Comment:
Should we add `default_value` for `oracle_protocol`?

Reply:
Good call. I've also added the `value` for oracle_home